cdap-proto/src/main/java/io/cdap/cdap/proto/app/AppUpdateGitMetaRequest.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright © 2023 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.cdap.proto.app;
+
+import java.util.Objects;
+
+public class AppUpdateGitMetaRequest {
+  private final String appId;
```

Comment:
make this consistent with ApplicationDetail.

appId -> name, fileHash -> sourceControlMeta

Reply:
done. Also renamed the class to `AppUpdateSourceControlMetaRequest`.